Model/Sales/TrackTraceHolder.php

```diff
@@ -194,8 +194,12 @@ public function convertDataFromMagentoToApi(Track $magentoTrack, array $options)
             ->setOnlyRecipient($this->getValueOfOption($options, 'only_recipient'))
             ->setSignature($this->getValueOfOption($options, 'signature'))
             ->setReturn($this->getValueOfOption($options, 'return'))
-            ->setLargeFormat($this->checkLargeFormat())
-            ->setAgeCheck($this->getAgeCheck($magentoTrack, $address))
+            ->setLargeFormat(
+                $this->getValueOfOptionWhenSet($options, 'large_format')
+                ?? $this->checkLargeFormat())
```

Comment:
Kunnen deze extra's niet net zo goed nu in `checkLargeFormat` en `getAgeCheck` zelf?